src/utils/recommendation.ts

```diff
@@ -161,21 +164,20 @@ export const generateUser = ({
   const [startHours, startMinutes] = startTime
     .split(":")
     .map((s) => _.toInteger(s));
-  const startDate = new Date(
-    Date.parse(
-      `2022-11-01T${startHours.toString().padStart(2, "0")}:${startMinutes
-        .toString()
-        .padStart(2, "0")}:00Z`
+
+  dayjs.extend(utc);
+  dayjs.extend(timezone);
+
+  const startDate = dayjs
+    .tz(
+      "2022-11-01 " + startHours + ":" + startMinutes + "00",
```

Comment:
use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)

Reply:
Seems like a nit, explain to me tomorrow why this is better, and what it would look like as a template literal. @lukejianu